### DISCUSSION TITLE
同一个配置文件，在同样meta内核,在Windows和安卓clash meta for Android 中都是走代理，可在openclash 里是直连

### DISCUSSION THREAD

**Discussion by liumingan** [vernesong/OpenClash#3555] [Q&A]
比如https://ipleak.net/，还有其他，不知道是什么原因，？？？？？
mixed-port: 7890
allow-lan: true
mode: Rule
log-level: info
external-controller: 0.0.0.0:9090
dns:
  enable: true
  ipv6: true
  fake-ip-range: 28.0.0.1/8
  fake-ip-filter:
    - '*'
    - '+.lan'
  hosts:
    - '*.clash.dev': 127.0.0.1
    - '.dev': 127.0.0.1
    - 'alpha.clash.dev': '::1' 
    - 'geosite:category-ads-all': 127.0.0.1
  default-nameserver:
   - 114.114.114.114
   - 119.29.29.29
   - tls://223.5.5.5:853
   - https://223.5.5.5/dns-query
  nameserver-policy:
    'ip.flares.cloud': '114.114.114.114'
    'geosite:baidu': '114.114.114.114'
    'geosite:alibaba': '223.5.5.5'
    'geosite:tencent': '119.29.29.29'
    'geosite:cn,private,category-games@cn,direct':
     - https://doh.pub/dns-query
     - https://dns.alidns.com/dns-query
     - 218.106.127.114 #联通
     - 218.104.111.114 #联通
     - "[2408:8888::8]:53" #联通
     - "[2408:8899::8]:53" #联通
    'rule-set:google,proxy,tld-not-cn,gfw':
     - 'tls://8.8.4.4:853#PROXY'
     - 'https://dns.google/dns-query#PROXY'
     - 'tls://1.0.0.1:853#PROXY'
     - 'https://cloudflare-dns.com/dns-query#PROXY'
    'geosite:geolocation-!cn,google,gfw,youtube':
     - 'tls://8.8.4.4:853#PROXY'
     - 'https://dns.google/dns-query#PROXY'
     - 'tls://1.0.0.1:853#PROXY'
     - 'https://cloudflare-dns.com/dns-query#PROXY'
  nameserver:
    - 218.106.127.114
    - 218.104.111.114
    - tls://dot.pub
    - https://dns.alidns.com/dns-query
  proxy-server-nameserver:
    - 'tls://1.0.0.1:853#WARP'
    - https://dns.alidns.com/dns-query
    - https://doh.pub/dns-query
  fallback:
    - 'tls://8.8.8.8:853#PROXY'
    - 'tls://1.0.0.1:853#PROXY'
    - 'https://cloudflare-dns.com/dns-query#PROXY'
    - 'https://dns.google/dns-query#PROXY'
    - 'https://mozilla.cloudflare-dns.com/dns-query#PROXY&h3=true'
  fallback-filter:
    geoip: true
    geoip-code: CN
    ipcidr:
    - 0.0.0.0/8
    - 10.0.0.0/8
    - 100.64.0.0/10
    - 127.0.0.0/8
    - 169.254.0.0/16
    - 172.16.0.0/12
    - 192.0.0.0/24
    - 192.0.2.0/24
    - 192.88.99.0/24
    - 192.168.0.0/16
    - 198.18.0.0/15
    - 198.51.100.0/24
    - 203.0.113.0/24
    - 224.0.0.0/4
    - 255.255.255.255/32
    domain:
    - "+.google.com"
    - "+.facebook.com"
    - "+.youtube.com"
    - "+.githubusercontent.com"
    - "+.googlevideo.com"
    - "+.msftconnecttest.com"
    - "+.msftncsi.com"
    - "+.v2ex.com"
    - "+.github.com"
tun:
  enable: true
  stack: gvisor
  auto-route: true
  auto-detect-interface: true
  dns-hijack:
    - any:53

rules:
  # 自定义规则
  - IP-CIDR,1.1.1.1/32,WARP,no-resolve
  - IP-CIDR,1.0.0.1/32,WARP,no-resolve
  - DOMAIN-SUFFIX,dyndns.com,DIRECT
  - DOMAIN-SUFFIX,dynv6.net,DIRECT
  - DOMAIN-SUFFIX,dynv6.com,DIRECT
  - GEOSITE,bing,PROXY

  # 规则集
  - RULE-SET,reject,REJECT
  - RULE-SET,google,google
  - RULE-SET,applications,DIRECT
  - RULE-SET,private,DIRECT
  - RULE-SET,direct,DIRECT
  - RULE-SET,icloud,Icloud
  - RULE-SET,apple,Apple
  - RULE-SET,gfw,PROXY
  - RULE-SET,proxy,PROXY
  - RULE-SET,tld-not-cn,PROXY
  - GEOSITE,apple-cn,Apple
  - GEOSITE,bilibili,DIRECT
  - GEOSITE,icloud,Icloud
  - GEOSITE,onedrive,Onedrive
  - GEOSITE,youtube,PROXY
  - GEOSITE,telegram,Telegram
  - GEOSITE,github,PROXY
  - GEOSITE,microsoft,Microsoft
  - GEOSITE,openai,OpenAI
  - GEOSITE,category-games@cn,DIRECT
  - GEOSITE,geolocation-!cn,PROXY
  - GEOSITE,CN,DIRECT
  - RULE-SET,lancidr,DIRECT,no-resolve
  - RULE-SET,telegramcidr,Telegram,no-resolve
  - RULE-SET,cncidr,DIRECT,no-resolve
  - GEOIP,LAN,DIRECT,no-resolve
  - GEOIP,GOOGLE,PROXY,no-resolve
  - GEOIP,telegram,Telegram,no-resolve
  - GEOIP,PRIVATE,DIRECT,no-resolve
  - GEOIP,CN,DIRECT,no-resolve
 - MATCH,Other,no-resolve

**Comment by liumingan** [comment]
改用tun模式

**Comment by vernesong** [comment]
看日志判断
